src/jit/SimdArm32Inl.h

```diff
@@ -296,43 +296,53 @@ static void F64x2PMax(double* src0, double* src1, double* dst)
     dst[1] = src0[1] < src1[1] ? src1[1] : src0[1];
 }
 
-static void simdEmitOp(sljit_compiler* compiler, uint32_t opcode, sljit_s32 vd, sljit_s32 vn, sljit_s32 vm)
-{
-    vd = sljit_get_register_index(SLJIT_SIMD_REG_64, vd);
-    vm = sljit_get_register_index(SLJIT_SIMD_REG_64, vm);
+#define highRegister 0x100
 
-    if (vn >= SLJIT_FR0) {
-        vn = sljit_get_register_index(SLJIT_SIMD_REG_64, vn);
+static sljit_s32 setRegister(sljit_s32 reg)
+{
+    sljit_s32 flags = reg & ~0xFF;
+    reg = sljit_get_register_index(SLJIT_SIMD_REG_64, reg & 0xFF);
+    if (flags & highRegister) {
+        return reg + (~reg & 0x1);
```

Comment:
Are these physical registers? `reg | 0x1` and `reg & ~0x1` should be easier. `?:` can be used as well

Reply:
After `reg = sljit_get_register_index(SLJIT_SIMD_REG_64, reg & 0xFF);` **reg** contains the number of a physical register.